modules/internal/ChapelDomain.chpl

```diff
@@ -1829,18 +1827,11 @@ module ChapelDomain {
      automatically commit indices to the sparse domain as the buffer fills up.
      Indices are also committed when the buffer goes out of scope.
 
-       .. note::
-
-          The interface and implementation are not stable and may change in the
-          future.
-
      :arg size: Size of the buffer in number of indices.
      :type size: int
     */
+    @unstable "makeIndexBuffer() is subject to change in the future."
```

Comment:
This function is also documented in the spec and so now has an unstable warning on it.